include/constants/dungeon.h

```diff
@@ -3,6 +3,9 @@
 
 #define NUM_DUNGEON_MAZE (NUM_DUNGEONS - DUNGEON_NORMAL_MAZE_2)
 
+// Not sure why I need to +1 the difference...
```

Comment:
Since both Steel Maze and Normal Maze are included in the count, it makes sense that you'd need a +1 here.